apps/src/aiTutor/views/teacherDashboard/InteractionsTable.tsx

```diff
@@ -106,43 +209,38 @@ const InteractionsTable: React.FC<InteractionsTableProps> = ({sectionId}) => {
     }
   };
 
-  const filteredChatMessages = chatMessages.filter(message => {
-    const statusMatch =
-      selectedStatuses.length === 0 ||
-      selectedStatuses.includes(message.status);
-
-    const userMatch =
-      selectedUserIds.length === 0 ||
-      selectedUserIds.includes(`${message.userId}`);
-
-    const messageDate = moment(message.createdAt);
-    let timeMatch = false;
+  const isTimeMatch = (createdAt: string) => {
```

Comment:
The time matching logic didn't change, but adding an `isTimeMatch` helper made the chat message filtering logic more readable.